float64[] t
float32[] joint_angles
float32[] joint_torques
